contributing.md

```diff
@@ -40,9 +40,10 @@ company_name
 }
 ```
 
-## Categories
-Here are the current categories we recognize. If your company category is not reflected here, feel free to use a new category. If you do so, kindly edit this file in your PR as well.
 
+4. If your company category is not reflected in categories section *below*, feel free to use a new category. If you do so, kindly edit this file (**/contributing.md**) in your PR as well.
+
```

Comment:
heyy @dbkonxepts, great work here! Can you please link to `contributing.md` directly here?

Reply:
Alright, coming through.

Reply:
Sorry, can you rephrase?
_Link to the same file?_

Reply:
Ah, seen. No need to link to this